### ISSUE TITLE
Returns false if cURL request fails

### ISSUE THREAD

**Issue by sapphyrus** [ron-maxweb/pwned-passwords#3]
How to reproduce:

1. Unset `curl.cainfo` in `php.ini` or change the API URL to something invalid

2. Run the following script
```
<?php

require_once('vendor/autoload.php');

$pp = new PwnedPasswords\PwnedPasswords;

// will return false
var_dump($pp->isInsecure('password'));

```

**Comment by mxrxdxn** [comment]
I was notified of this a little while ago - I'm in the process of handling curl failures, expect to see a fix in the next couple days.

**Comment by azjezz** [comment]
i tried fixing this by throwing `RuntimeException` if curl/file_get_contents failed.
you can try to catch it.
https://github.com/ron-maxweb/pwned-passwords/pull/4#issuecomment-373878269

**Comment by azjezz** [comment]
@ron-maxweb i think you should close this issues as it was fixed in my last PR by throwing `RuntimeException`

**Comment by mxrxdxn** [comment]
Fixed in #4.
